src/components/activity/FormInput.js

```diff
@@ -0,0 +1,49 @@
+import {useState} from 'react';
+import {FormStyles} from './styles/FormInput.styles';
+import {nanoid} from 'nanoid';
+
+function FormInput({addActivity}) {
+  const [activity, setActivity] = useState({
+    id: nanoid(),
+    activity: '',
```

Comment:
Maybe this will be a better naming? 
```suggestion
    activityName: '',
```